templates/modelEnum.mustache

```diff
@@ -76,6 +77,11 @@ import com.google.gson.stream.JsonWriter;
       return {{{datatypeWithEnum}}}{{^datatypeWithEnum}}{{{classname}}}{{/datatypeWithEnum}}.fromValue({{#isNumber}}new BigDecimal({{/isNumber}}value{{#isNumber}}){{/isNumber}});
     }
   }
+
+  public static void validateJsonElement(JsonElement jsonElement) throws IOException {
```

Comment:
This is largely a copy-paste of the `read` function above, modified to work with a `JsonElement` rather than a `JsonReader`: https://github.com/equinix-labs/metal-java/pull/163/commits/ddc37db375e481dfd1b2c317d30886dbc684a419#diff-e633e9a83ea415a1143582253f12897bcac9cda46fe610c6145443ab6e14ed2dR75-R78